### PR TITLE
Bump Julia lower bound

### PR DESCRIPTION
**Pull request by odow**
MathOptInterface has a lower bound of 1.6 so you cannot install on pre 1.6

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -8,12 +8,11 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 PrecompileTools = "aea7be01-6a6a-4083-8856-8a6e6704d82a"
 TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
-#Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 MathOptInterface = "1"
 PrecompileTools = "1"
-julia = "1.8"
+julia = "1.6"
```
Comment (odow) [inline Project.toml:15]: Why'd you bump the minimum Julia version to 1.8? The current long-term support is Julia v1.6, so unless you're using some new features, things should just work.

Comment (PharosAbad) [reply Project.toml:15]: [Problem-Algorithm-Solver pattern](https://discourse.julialang.org/t/function-depending-on-the-global-variable-inside-module/64322/10) mention that 1.7 is needed, and I code the project in 1.8,  the CI checks on 1.8 and 1.9

Comment (odow) [reply Project.toml:15]: I changed the code to test on 1.6.

Comment (PharosAbad) [reply Project.toml:15]: 1.6 :

 ```
ERROR: LoadError: LoadError: syntax: invalid assignment location "; V, A, G, b, g, d, u, N, M, J" around /home/runner/work/StatusSwitchingQP.jl/StatusSwitchingQP.jl/src/types.jl:213
```

**Comment by odow** [comment]
Ah, so you're using some post Julia 1.6 syntax fo
https://github.com/PharosAbad/StatusSwitchingQP.jl/blob/642bd04e815b9edfd702a8aa4d9d41d0b3ec100c/src/types.jl#L213

**Comment by odow** [comment]
I changed the syntax. Unless there's a very good reason, you should try to stick with the Julia 1.6 syntax. otherwise we'll need to put a note in the JuMP documentation that this solver is only supported on some versions.

**Pull request closed without merging**